linux_os/guide/services/ntp/chronyd_server_directive/rule.yml

```diff
@@ -0,0 +1,30 @@
+documentation_complete: true
+
+title: 'Ensure Chrony is only configured with the server directive'
+
+description: |-
+    Check that Chrony only has time sources configured with the <tt>server</tt> directive.
+
+rationale: |-
+    Depending on the infrastruture being used the <tt>pool</tt> directive may not be supported.
+
+severity: medium
+
+platform: chrony
+
+warnings:
+  - general: This rule doesn't come with a remediation, the time source needs to be added by the adminstrator.
+
+identifiers:
+    cce@rhel8: CCE-86077-5
+
+references:
+    disa: CCI-001891
+    stigid@rhel8: RHEL-08-030740
```

Comment:
Please add CCE for RHEL9 and select this new rule in the RHEL9 STIG profile as well.

Also add the SRG references from https://vaulted.io/library/disa-stigs-srgs/red_hat_enterprise_linux_8_security_technical_implementation_guide/V-230484
`
Satisfies: SRG-OS-000355-GPOS-00143, SRG-OS-000356-GPOS-00144, SRG-OS-000359-GPOS-00146`